actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   `remote_ip` will no longer ignore IPs in X-Forwarded-For headers if they
+    are accompanied by port information.
+
+    *Duncan Brown*, *Provenios Marinos*
```

Comment:
@duncanjbrown Thanks for the mention but the name it's Prevenios Marinos :)

Reply:
Argh! Sorry — amended.